packaging/package/jar_jdk_packager.py

```diff
@@ -2,15 +2,63 @@
 import logging
 import subprocess
 import shutil
+import xml.etree.ElementTree as ET
 
 from pathlib import Path
 from .connector_file import ConnectorFile
 from .helper import check_jdk_environ_variable
+from .version import __min_version_tableau__
 
 JAR_EXECUTABLE_NAME = "jar.exe"
+MANIFEST_FILE_TYPE = "manifest"
+MANIFEST_FILE_NAME = "manifest.xml"
+MANIFEST_ROOT_ELEM = "connector-plugin"
+MANIFEST_FILE_COPY_NAME = "manifest_copy.xml"
+MIN_TABLEAU_VERSION_ATTR = "min-version-tableau"
 logger = logging.getLogger(__name__)
 
 
+def stamp_min_support_version(input_dir, file_list):
+    """
+    Stamp minimum support version on manifest file,
+    make a copy of manifest to recover int the future
+
+    :param input_dir: source dir of files to be packaged
+    :type input_dir: str
+
+    :param file_list: files need to be packaged
+    :type file_list: list of ConnectorFile
+
+    :return: Boolean
+    """
+
+    # find manifest in connector file list
+    manifest_file = None
+    for file in file_list:
+        if file.file_type == MANIFEST_FILE_TYPE and file.file_name == MANIFEST_FILE_NAME:
+            manifest_file = file
```

Comment:
should we add a `break` here since there should only be one `ConnectorFile` obj that corresponds to a `manifest_file`?
```suggestion
            manifest_file = file
            break
```

Reply:
Thanks, done.